pkg/hintrunner/zero/zerohint_uint256_test.go

```diff
@@ -0,0 +1,87 @@
+package zero
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+	// "github.com/NethermindEth/cairo-vm-go/pkg/utils"
```

Comment:
A non-blocking nit: this can be removed later if it's easier to merge right away.

Reply:
I was using it at some point, then it wasn't necessary anymore but thought I would have to use it again. Of course, I ended up forgetting about it.